### PR TITLE
[102] Draft

### PR DESCRIPTION
**Pull request by tapthaker**
#361

### CODE REVIEW

File: _drafts/2018-02-08-issue-102.md [hunk 1]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
...
 
 ### Swift Unwrapped
 
-> TODO: Latest episode of Swift Unwrapped
+[Swift Bi-Weekly Brief](https://spec.fm/podcasts/swift-unwrapped/108765)
 
 ### News and community
 
-> TODO
+[Greg Heo](https://twitter.com/gregheo) wrote about an amazing post about [Conditional conformance](https://swiftunboxed.com/lang/conditional-conformance/) Thinking through conditional conformance in Swift, and working through the basics of its implementation.
```
Comment (RomanVolkov) [inline _drafts/2018-02-08-issue-102.md]: doubled "about"

File: _drafts/2018-02-08-issue-102.md [hunk 2]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
...
+
+Xi Ge's talk about [Creating Refactoring Transformations for Swift](https://www.skilled.io/u/swiftsummit/creating-refactoring-transformations-for-swift) from the [Swift Summit 2017](https://www.swiftsummit.com/) is posted. He also talks about future of refactoring engine with [libSyntax](https://github.com/apple/swift/blob/master/lib/Syntax/README.md).
```
Comment (RomanVolkov) [inline _drafts/2018-02-08-issue-102.md]: Xi Ge's profile: https://github.com/nkcsgexi

File: _drafts/2018-02-08-issue-102.md [hunk 3]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
...
+[Peter Steinberger](https://twitter.com/steipete) wrote about [Binary Frameworks in Swift](https://pspdfkit.com/blog/2018/binary-frameworks-swift/)
+>This article explores what ABI means and how it can be important for third-party frameworks.
+> <br/>TL;WR: ABI stability won’t change much for you, and it’s not enough to ship binary Swift frameworks.
+
+
 
 ### Commits and pull requests
 
-> TODO
+Doug Gregor [fixed](https://github.com/apple/swift/pull/14174) an [issue](https://bugs.swift.org/browse/SR-6841) in type checker where class constraints `(spelled T: AnyObject)` on generic types were not getting checked on generic arguments.
```
Comment (RomanVolkov) [inline _drafts/2018-02-08-issue-102.md]: It will be nice to place links to github/twitter profiles.

Comment (jessesquires) [reply _drafts/2018-02-08-issue-102.md]: hey @RomanVolkov would you be interested in working on #316? it would make doing this much easier 😊

Comment (RomanVolkov) [reply _drafts/2018-02-08-issue-102.md]: hey @jessesquires Yup, I'll take a look on weekend:)

File: _drafts/2018-02-08-issue-102.md [hunk 4]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
 
 > TODO: intro comments
 
-<!--excerpt-->
```
Comment (jessesquires) [inline _drafts/2018-02-08-issue-102.md]: this needs to stay

Comment (jessesquires) [reply _drafts/2018-02-08-issue-102.md]: this is part of how jekyll builds the site. the snippet before this is use in the RSS/JSON feed as the "summary"

File: _drafts/2018-02-08-issue-102.md [hunk 5]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
...
+- [SR-6852](https://bugs.swift.org/browse/SR-6852) Support `#if swift(<4.1)` *#compiler*
+- [SR-6808](https://bugs.swift.org/browse/SR-6808) Visualizing Build Graph is not exposed from llbuild Build System *#llbuild*
+- [SR-6787](https://bugs.swift.org/browse/SR-6787) Unexpected result when getting a String describing a type created inside a function *#bug* *#standard-library*
 
 {% include task_remind.html %}
 
 ### Swift Unwrapped
 
-> TODO: Latest episode of Swift Unwrapped
+[Swift Bi-Weekly Brief](https://spec.fm/podcasts/swift-unwrapped/108765)
```
Comment (jessesquires) [inline _drafts/2018-02-08-issue-102.md]: we'll need to add a description here 😊

File: _drafts/2018-02-08-issue-102.md (hunk 1 above)
Comment (jessesquires) [inline _drafts/2018-02-08-issue-102.md]: i think this sentence is incomplete. how about:

> [Greg Heo](https://twitter.com/gregheo) wrote about an amazing post about [Conditional conformance](https://swiftunboxed.com/lang/conditional-conformance/). In it, he thinks through conditional conformance in Swift, and works through the basics of its implementation.

File: _drafts/2018-02-08-issue-102.md [hunk 6]
```diff
@@ -6,25 +6,46 @@ author: tapthaker
...
 
 ### News and community
 
-> TODO
+[Greg Heo](https://twitter.com/gregheo) wrote about an amazing post about [Conditional conformance](https://swiftunboxed.com/lang/conditional-conformance/) Thinking through conditional conformance in Swift, and working through the basics of its implementation.
+
+Xi Ge's talk about [Creating Refactoring Transformations for Swift](https://www.skilled.io/u/swiftsummit/creating-refactoring-transformations-for-swift) from the [Swift Summit 2017](https://www.swiftsummit.com/) is posted. He also talks about future of refactoring engine with [libSyntax](https://github.com/apple/swift/blob/master/lib/Syntax/README.md).
+
+[Brandon Williams](http://www.fewbutripe.com/about/) and [Stephen Celis](http://www.stephencelis.com/) launched a new video series called [Point-Free](https://www.pointfree.co/). Checkout their first episode [Functions](https://www.pointfree.co/episodes/ep1-functions).
```
Comment (jessesquires) [inline _drafts/2018-02-08-issue-102.md]: nit: *a new Swift video series* 😄 

comma after "Checkout their first episode" --> "Checkout their first episode, "

File: _drafts/2018-02-08-issue-102.md [hunk 7]
```diff
@@ -40,7 +61,9 @@ author: tapthaker
 
 ### Proposals in review
 
-> TODO
+[SE-0197](https://github.com/apple/swift-evolution/blob/master/proposals/0197-remove-where.md): Adding in-place `remove(where:)` to the Standard Library is in review.
```
Comment (jessesquires) [inline _drafts/2018-02-08-issue-102.md]: link "in review" to the Swift forum review 😊

**Comment by tapthaker** [comment]
Sure. Will do that.

File: _drafts/2018-02-08-issue-102.md (hunk 1 above)
Comment (SD10) [reply _drafts/2018-02-08-issue-102.md]: Just make sure we remove this double "about":
Greg Heo wrote about an amazing post about Conditional conformance.
This grammar states that Greg Heo wrote a post about a post 😅

Comment (BasThomas) [reply _drafts/2018-02-08-issue-102.md]: Was already noted [here](https://github.com/SwiftWeekly/swiftweekly.github.io/pull/363#discussion_r164969856), but yeah :)

File: _drafts/2018-02-08-issue-102.md (hunk 4 above)
Comment (tapthaker) [reply _drafts/2018-02-08-issue-102.md]: Ok, thanks.

File: _drafts/2018-02-08-issue-102.md (hunk 5 above)
Comment (tapthaker) [reply _drafts/2018-02-08-issue-102.md]: Yes, listening to the episode is still pending on my side. Will add a description once I finish listening to it.

**Comment by tapthaker** [comment]
@jessesquires @BasThomas @RomanVolkov : Will be closing this PR soon and creating a new PR for upcoming changes.

**Comment by tapthaker** [comment]
The links to author names & the fix looks odd here. It looks as if it is the same link. Any suggestions ?
screen shot 2018-02-04 at 6 11 37 am

**Comment by jessesquires** [comment]
@tapthaker No need to close this. We can merge, since it's only in `_drafts/`. I'll do that now.

**Pull request merged**